src/resources/lib/modules/services.py

```diff
@@ -341,6 +361,7 @@ def load_values(self):
                 self.struct['ssh']['settings']['ssh_autostart']['value'] = self.oe.get_service_state('sshd')
                 self.struct['ssh']['settings']['ssh_secure']['value'] = self.oe.get_service_option('sshd', 'SSHD_DISABLE_PW_AUTH',
                         self.D_SSH_DISABLE_PW_AUTH).replace('true', '1').replace('false', '0').replace('"', '')
+                self.struct['ssh']['settings']['ssh_passwd']['value'] = 'libreelec'
```

Comment:
Can we leave this value blank? We shouldn't be encoding any default password, as there's no need for this value, right?

Reply:
This value can come out - it is from the previous version that contained the view of the password